eo-runtime/src/main/java/org/eolang/BytesOf.java

```diff
@@ -142,7 +142,7 @@ public Bytes shift(final int bits) {
                 } else {
                     byte dst = (byte) (bytes[source] << mod);
                     if (source + 1 < bytes.length) {
-                        dst |= bytes[source + 1] >>> (Byte.SIZE - mod) & carry & 0xFF;
+                        dst |= bytes[source + 1] >>> (Byte.SIZE - mod) & (carry & 0xFF) & 0xFF;
```

Comment:
@volodya-lombrozo I believe just adding braces would do the trick. No need in second `& 0xFF`

Reply:
@mximp You right. Thanks.